### ISSUE TITLE
Distribution configuration should be exposed

### ISSUE THREAD

**Issue by cnelson** [18F/cf-cdn-service-broker#41]
For compliance reasons it's important for users of the broker to be able to inspect the distribution configuration so they can ensure it is in a known-good-state.

The broker should expose the output of `cloudfront:GetDistribution` whenever a configuration change is made.

**Comment by afeld** [comment]
Potentially part of https://github.com/18F/cf-cdn-service-broker/issues/50.

**Comment by mogul** [comment]
@cnelson says this came from a support inquiry on behalf of FEC; check if it's needed to enable their move.

**Comment by cnelson** [comment]
Federalist not FEC 

cc: @wslack

**Comment by wslack** [comment]
Yep. We need it for ATO purposes (so we can verify we config'd cloud.gov correctly).

**Comment by wslack** [comment]
Wait @cnelson that means this is done, correct?

**Comment by mogul** [comment]
@cnelson can you confirm?

**Comment by cnelson** [comment]
No, nothing was done on this.  I was just closing ancient issues and pull requests that won't  see any attention.  if y'all still want this, I'd open a favro card to track this feature request.

**Comment by wslack** [comment]
please do!
